src/domain/404_page/err404.module.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../utils/variables';
+
+.box{
+    background: $lightGrey;
+    position: relative;
+    display: block;
+    text-align: center;
+    object-fit: fill;
+    padding: 1em;
+    & img {
+        text-align: center;
+        width: 100%;
+        max-width: 800px;
+    }
+}
+
+.text{
+    position: absolute;
+    left: 48%;
+    top: 10%;
+    text-align: left;
+    & h2 {
```

Comment:
Pretty sure the '&'s are not required throughout

```
.text{
  & h2 {}
}

```
is identical to the standard 

```
.text{
  h2 {}
}
```

(This would not be the case if there was no space)

If you care:
https://css-tricks.com/the-sass-ampersand/#the-aha-moment